images/parity_parity/src/image.rs

```diff
@@ -0,0 +1,65 @@
+pub struct ParityEthereum {
+    arguments: ParityArgs,
+}
+
+#[derive(Clone)]
+pub struct ParityArgs {
```

Comment:
The convention that we followed so far was `<ImageName>ImageArgs`, so this would be `ParityEthereumImageArgs`.

Although the docker image is named `parity/parity`, I think it would be better to name the whole crate `tc_parity_parityethereum` to make it more clear. They also renamed the github repository recently, so it is probably just a matter of time until the rename the docker image aswell.